### ISSUE TITLE
Add a loader

### ISSUE THREAD

**Issue by run1t** [tinderjs/tinder-desktop#99]
We should add loader in order to wait at least the response of the getAccount() method.

**Comment by hanford** [comment]
@run1t agreed, Can I take this one? I want to integrate a youtube-esque loader 😎

**Comment by run1t** [comment]
@hanford  Yes of course ;)
